### PR TITLE
refactor part1 change define in coulours.dm from COULOUR to COLOR

### PR DESCRIPTION
**Pull request by Huffie56**
# About the pull request

So the idea is to standartize and remove as much as possible the use of colour.
in this first PR i wanted to start with define using colour in them.

i also stole define in _math.dm because they add a bunch of define that would have been in conflict. and i didn't really understand why some color where define there...

can understand for blood and pipe but why math would need colors defines??

# Explain why it's good for the game
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
refactor: for the file coulours.dm and other files change define in it from COULOUR to COLOR
/:cl:

### CODE REVIEW

**Comment by Huffie56** [comment]
need a break will write about the pull request after if test go green.

File: code/__DEFINES/colours.dm [hunk 1]
```diff
@@ -1,214 +1,228 @@
...
+#define CL_COLOR_VERY_LIGHT_GRAY "#EEEEEE"
+#define CL_COLOR_SILVER "#C0C0C0"
+#define CL_COLOR_GRAY "#808080"
+#define CL_COLOR_FLOORTILE_GRAY "#8D8B8B"
+#define CL_COLOR_DARK "#454545"
+#define CL_COLOR_ALMOST_BLACK "#333333"
+#define CL_COLOR_BLACK "#000000"
+#define CL_COLOR_HALF_TRANSPARENT_BLACK "#0000007A"
+
+#define CL_COLOR_RED "#FF0000"
```
Comment (realforest2001) [inline code/__DEFINES/colours.dm:29]: Duplicate define of COLOR_RED, can remove entirely.

File: code/__DEFINES/colours.dm [hunk 2]
```diff
@@ -1,214 +1,228 @@
...
-
-///Colours for eigenstates
-#define COLOUR_PERIWINKLEE "#9999FF"
+#define CL_COLOR_PRIORITY_AMOUNT 4
+
+#define CL_COLOR_DARKMODE_BACKGROUND "#202020"
+#define CL_COLOR_DARKMODE_DARKBACKGROUND "#171717"
+#define CL_COLOR_DARKMODE_TEXT "#a4bad6"
+
+#define CL_COLOR_WHITE "#FFFFFF"
```
Comment (realforest2001) [inline code/__DEFINES/colours.dm:19]: Duplicate define of COLOR_WHITE, can remove entirely.

File: code/__DEFINES/colours.dm [hunk 3]
```diff
@@ -1,214 +1,228 @@
...
+#define CL_COLOR_VERY_LIGHT_GRAY "#EEEEEE"
+#define CL_COLOR_SILVER "#C0C0C0"
+#define CL_COLOR_GRAY "#808080"
+#define CL_COLOR_FLOORTILE_GRAY "#8D8B8B"
+#define CL_COLOR_DARK "#454545"
+#define CL_COLOR_ALMOST_BLACK "#333333"
+#define CL_COLOR_BLACK "#000000"
```
Comment (realforest2001) [inline code/__DEFINES/colours.dm:26]: Same, COLOR_BLACK

File: code/__DEFINES/colours.dm [hunk 4]
```diff
@@ -1,214 +1,228 @@
...
+#define CL_COLOR_SOFT_RED "#FA8282"
+#define CL_COLOR_CULT_RED "#960000"
+#define CL_COLOR_BUBBLEGUM_RED "#950A0A"
+
+#define CL_COLOR_YELLOW "#FFFF00"
+#define CL_COLOR_VIVID_YELLOW "#FBFF23"
+#define CL_COLOR_VERY_SOFT_YELLOW "#FAE48E"
+
+#define CL_COLOR_OLIVE "#808000"
+#define CL_COLOR_VIBRANT_LIME "#00FF00"
```
Comment (realforest2001) [inline code/__DEFINES/colours.dm:45]: VIBRANT_LIME, duplicate of existing COLOR_GREEN

File: code/__DEFINES/colours.dm [hunk 5]
```diff
@@ -1,214 +1,228 @@
...
+#define CL_COLOR_LIME "#32CD32"
+#define CL_COLOR_DARK_LIME "#00aa00"
+#define CL_COLOR_VERY_PALE_LIME_GREEN "#DDFFD3"
+#define CL_COLOR_VERY_DARK_LIME_GREEN "#003300"
+#define CL_COLOR_GREEN "#008000"
+#define CL_COLOR_DARK_MODERATE_LIME_GREEN "#44964A"
+
+#define CL_COLOR_CYAN "#00FFFF"
```
Comment (realforest2001) [inline code/__DEFINES/colours.dm:53]: Duplicate

File: code/__DEFINES/colours.dm [hunk 6]
```diff
@@ -1,214 +1,228 @@
...
+#define CL_COLOR_DARK_CYAN "#00A2FF"
+#define CL_COLOR_TEAL "#008080"
+#define CL_COLOR_BLUE "#0000FF"
```
Comment (realforest2001) [inline code/__DEFINES/colours.dm:56]: Duplicate

File: code/__DEFINES/colours.dm [hunk 7]
```diff
@@ -1,214 +1,228 @@
...
+#define CL_COLOR_AMETHYST "#822BFF"
+#define CL_COLOR_BLUE_LIGHT "#33CCFF"
+#define CL_COLOR_NAVY "#000080"
+#define CL_COLOR_BLUE_GRAY "#75A2BB"
+
+#define CL_COLOR_PINK "#FFC0CB"
+#define CL_COLOR_LIGHT_PINK "#ff3cc8"
+#define CL_COLOR_MOSTLY_PURE_PINK "#E4005B"
+#define CL_COLOR_BLUSH_PINK "#DE5D83"
+#define CL_COLOR_MAGENTA "#FF00FF"
```
Comment (realforest2001) [inline code/__DEFINES/colours.dm:69]: COLOR_MAGENTA, duplicate of existing COLOR_PINK

File: code/__DEFINES/colours.dm [hunk 8]
```diff
@@ -1,214 +1,228 @@
...
+#define CL_COLOR_STRONG_MAGENTA "#B800B8"
+#define CL_COLOR_PURPLE "#800080"
+#define CL_COLOR_VIOLET "#B900F7"
+#define CL_COLOR_STRONG_VIOLET "#6927c5"
+
+#define CL_COLOR_ORANGE "#FF9900"
```
Comment (realforest2001) [inline code/__DEFINES/colours.dm:75]: Duplicate

File: code/__DEFINES/colours.dm [hunk 9]
```diff
@@ -1,214 +1,228 @@
...
+#define CL_COLOR_ASSEMBLY_LBLUE   "#5D99BE"
+#define CL_COLOR_ASSEMBLY_BLUE "#38559E"
+#define CL_COLOR_ASSEMBLY_PURPLE  "#6F6192"
+
+///Colors for xenobiology vatgrowing
+#define CL_COLOR_SAMPLE_YELLOW "#c0b823"
+#define CL_COLOR_SAMPLE_PURPLE "#342941"
+#define CL_COLOR_SAMPLE_GREEN "#98b944"
+#define CL_COLOR_SAMPLE_BROWN "#91542d"
+#define CL_COLOR_SAMPLE_GRAY "#5e5856"
```
Comment (realforest2001) [inline code/__DEFINES/colours.dm:127]: Unused in code

File: code/__DEFINES/colours.dm [hunk 10]
```diff
@@ -1,214 +1,228 @@
...
+#define CL_COLOR_ASSEMBLY_BEIGE   "#AF9366"
+#define CL_COLOR_ASSEMBLY_BROWN   "#97670E"
+#define CL_COLOR_ASSEMBLY_GOLD "#AA9100"
+#define CL_COLOR_ASSEMBLY_YELLOW  "#CECA2B"
+#define CL_COLOR_ASSEMBLY_GURKHA  "#999875"
+#define CL_COLOR_ASSEMBLY_LGREEN  "#789876"
+#define CL_COLOR_ASSEMBLY_GREEN   "#44843C"
+#define CL_COLOR_ASSEMBLY_LBLUE   "#5D99BE"
+#define CL_COLOR_ASSEMBLY_BLUE "#38559E"
+#define CL_COLOR_ASSEMBLY_PURPLE  "#6F6192"
```
Comment (realforest2001) [inline code/__DEFINES/colours.dm:120]: Unused so far as I can see

File: code/__DEFINES/colours.dm [hunk 11]
```diff
@@ -1,214 +1,228 @@
...
+#define CL_COLOR_DARK_BROWN "#997C4F"
+#define CL_COLOR_ORANGE_BROWN "#a9734f"
+
+//Color defines used by the soapstone (based on readability against grey tiles)
+#define CL_COLOR_SOAPSTONE_PLASTIC "#a19d94"
+#define CL_COLOR_SOAPSTONE_IRON "#b2b2b2"
+#define CL_COLOR_SOAPSTONE_BRONZE "#FE8001"
+#define CL_COLOR_SOAPSTONE_SILVER "#FFFFFF"
+#define CL_COLOR_SOAPSTONE_GOLD "#FFD900"
+#define CL_COLOR_SOAPSTONE_DIAMOND "#00ffee"
```
Comment (realforest2001) [inline code/__DEFINES/colours.dm:95]: Unused

File: code/__DEFINES/colours.dm [hunk 12]
```diff
@@ -1,214 +1,228 @@
...
+
+#define CL_COLOR_GREEN_GRAY    "#99BB76"
+#define CL_COLOR_RED_GRAY  "#B4696A"
+#define CL_COLOR_PALE_BLUE_GRAY   "#98C5DF"
+#define CL_COLOR_PALE_GREEN_GRAY  "#B7D993"
+#define CL_COLOR_PALE_RED_GRAY "#D59998"
+#define CL_COLOR_PALE_PURPLE_GRAY "#CBB1CA"
+#define CL_COLOR_PURPLE_GRAY   "#AE8CA8"
```
Comment (realforest2001) [inline code/__DEFINES/colours.dm:103]: Unused

**Review by realforest2001 — CHANGES_REQUESTED** [review]
Looks like a majority of these defines aren't used anywhere so are safe to remove. A fair portion are also duplicate results of existing COLOR defines, so you can swap them out entirely.

**Comment by Huffie56** [comment]
> Looks like a majority of these defines aren't used anywhere so are safe to remove. A fair portion are also duplicate results of existing COLOR defines, so you can swap them out entirely.

so i can safely move every color define into colours.dm file?
i think it doesn't matter where define are made but i am not sure.... (didn't wanted to cause any issues...)

File: code/__DEFINES/colours.dm (hunk 1 above)
Comment (Huffie56) [reply code/__DEFINES/colours.dm:29]: yes i see what i need to do now after some sleep and your help.

File: code/__DEFINES/colours.dm (hunk 3 above)
Comment (Huffie56) [reply code/__DEFINES/colours.dm:26]: removed duplicate.

File: code/__DEFINES/colours.dm (hunk 4 above)
Comment (Huffie56) [reply code/__DEFINES/colours.dm:45]: isn't used and is duplicated deleted.

File: code/__DEFINES/colours.dm (hunk 5 above)
Comment (Huffie56) [reply code/__DEFINES/colours.dm:53]: yep deleted

File: code/__DEFINES/colours.dm (hunk 7 above)
Comment (Huffie56) [reply code/__DEFINES/colours.dm:69]: yep deleted

File: code/__DEFINES/colours.dm (hunk 12 above)
Comment (Huffie56) [reply code/__DEFINES/colours.dm:103]: yep not use so deleting

File: code/__DEFINES/colours.dm (hunk 11 above)
Comment (Huffie56) [reply code/__DEFINES/colours.dm:95]: yep

File: code/__DEFINES/colours.dm (hunk 10 above)
Comment (Huffie56) [reply code/__DEFINES/colours.dm:120]: yep cleaning stuff up

File: code/__DEFINES/colours.dm (hunk 6 above)
Comment (Huffie56) [reply code/__DEFINES/colours.dm:56]: cleaning stuff

File: code/__DEFINES/colours.dm (hunk 8 above)
Comment (Huffie56) [reply code/__DEFINES/colours.dm:75]: deleting stuff not use and duplicate

File: code/__DEFINES/colours.dm (hunk 9 above)
Comment (Huffie56) [reply code/__DEFINES/colours.dm:127]: yep deleting

**Review by realforest2001 — APPROVED** [review]

**Review by realforest2001 — APPROVED** [review]

**Review by Birdtalon — APPROVED** [review]
Fine.

**Pull request merged**